project-keeper/src/main/java/com/exasol/projectkeeper/validators/finding/SimpleValidationFinding.java

```diff
@@ -1,29 +1,52 @@
 package com.exasol.projectkeeper.validators.finding;
 
+import static java.util.function.Predicate.not;
+
+import java.util.List;
 import java.util.Objects;
+import java.util.stream.Collectors;
 
 import com.exasol.projectkeeper.Logger;
+import com.exasol.projectkeeper.ProjectKeeper;
 
-import lombok.EqualsAndHashCode;
-import lombok.ToString;
+import lombok.*;
+import lombok.experimental.Accessors;
 
 /**
  * This class represents an error that was found during validation.
  */
 @EqualsAndHashCode
 @ToString
 public class SimpleValidationFinding implements ValidationFinding {
+
+    /**
+     * Returns mandatory findings.
+     *
+     * @param findings list of finding to be filtered
+     * @return mandatory findings
+     */
+    public static List<SimpleValidationFinding> blockers(final List<SimpleValidationFinding> findings) {
+        return findings.stream() //
+                .filter(not(SimpleValidationFinding::isOptional)) //
+                .collect(Collectors.toList());
+    }
+
     private final String message;
+    @Getter
+    @Accessors(fluent = true)
```

Comment:
Check if this makes sense

Reply:
yes makes sense, but is redundant.
> fluent -- If true, accessors will be named after the field and not include a get or setprefix.

Default getter generated by lombok is already "isOptional()".